tools/build_blas.sh

```diff
@@ -11,7 +11,7 @@
 # usage: ./build_blas.sh src_root ndk_root <install_path>
 # if install path is not given, it is installed in $(pwd)
 # src should be prepared before running this script
-# this script is competible with https://github.com/xianyi/OpenBLAS/tree/v0.2.20
+# this script is competible with https://github.com/xianyi/OpenBLAS/tree/v0.3.18 or v0.2.20
```

Comment:
For now, we have observed performance degradation with v0.3.18 with this script.

There might be more flags or options to be added when building v0.3.18.

Reply:
revisit: Check that the issue was independent of cblas version update. It was system issue. Rather, there is a slight change that v0.3 is giving better parallelization.